### DISCUSSION TITLE
Is possible to hiden the username and password from the value.yaml file?

### DISCUSSION THREAD

**Discussion by mr11112121212313s** [airflow-helm/charts#751] [Questions & Answers]
Hi community, 

I have deployed the openmetadata to Amazon EKS and the username and password have leaked on the value.yaml file(https://github.com/airflow-helm/charts/blob/main/charts/airflow/values.yaml#L74), is that possible to hide the credentials by reference from the EKS cluster?

for example , i like to achieve something like the below, not sure if an airflow helm-chart will be supported. Thank you 

      users:
        - username:
             secretRef: xxxxx
             secretKey: airflow-username-secret
          password: 
             secretRef: xxxxx
             secretKey: airflow-password-secret
          role: Admin
          email: xxxxx
          firstName: xxx
          lastName: x

**Comment by HakarK** [comment]
You can add a configmap or a secret under usersTemplates, and refer to it like this:
```
  users:
    - username: ${USERNAME}
      password: ${PASSWORD}
```
      
```
## ____ EXAMPLE _______________
  ##   usersTemplates
  ##     MY_USERNAME:
  ##       kind: configmap
  ##       name: my-configmap
  ##       key: username
  ##     MY_PASSWORD:
  ##       kind: secret
  ##       name: my-secret
  ##       key: password
  ##
```

**Comment by thesuperzapper** [comment]
@mr11112121212313s as suggested by @HakarK, you can use the `usersTemplates` feature, for more information, see the following docs:
- https://github.com/airflow-helm/charts/blob/main/charts/airflow/docs/faq/security/airflow-users.md#define-with-templates-from-secrets-or-configmaps
